src/core/edit/edit.cell.view.js

```diff
@@ -6,7 +6,8 @@ import { getFactory as valueFactory } from '../services/value';
 import { getFactory as labelFactory } from '../services/label';
 import { parseFactory } from '../services/convert';
 import { Td } from '../dom/td';
-import * as validationService from '../validation/validation.service';
+import * as validationService from '../validation/validator.builder';
+import {ValidatorBuilder} from "../validation/validator.builder";
```

Comment:
Actually I don't know why it looks like this in github, because in code I have spaces around:
![image](https://user-images.githubusercontent.com/9286878/74331367-9c458b00-4da4-11ea-86d7-b6a786527c28.png)